src/topBar/TopBar.tsx

```diff
@@ -1,34 +1,25 @@
-import React, { Fragment, ReactNode } from 'react'
+import React, { ReactNode } from 'react'
 import cc from 'classcat'
 
 import { StyledTopBar } from './TopBar.style'
 
 export type TopBarProps = Readonly<{
   className?: string
+  zIndex: number
   leftItem?: ReactNode
   rightItem?: ReactNode
   centerItem?: ReactNode
-  fixed?: boolean
-  bgTransparent?: boolean
-  bgShadedTransparent?: boolean
-  innerWrapperClassName?: string
+  hasScrolled?: boolean
```

Comment:
I've added this `hasScrolled` prop to bring the "u-separator-bottom" style to ui-library.